packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.js

```diff
@@ -22,11 +22,15 @@ import React from 'react';
 import { cssModules } from 'bpk-react-utils';
 import BpkText from 'bpk-component-text';
 import BpkButton from 'bpk-component-button';
-import BpkCard from 'bpk-component-card';
 
-import STYLES from './BpkGraphicPromo.module.scss';
+import CARD_STYLES from '../../bpk-component-card/src/BpkCard.module.scss';
 
-const getClassName = cssModules(STYLES);
+import GP_STYLES from './BpkGraphicPromo.module.scss';
+
+const getClassName = cssModules(GP_STYLES);
+const getCardClassName = cssModules(CARD_STYLES);
+
+const ACCESSIBILITY_KEYS = [13 /* Enter */, 32 /* Space */];
```

Comment:
Can we turn this into an object instead similar to other components, this then avoids the need for a comment and lays out what they are

Example: https://github.com/Skyscanner/backpack/blob/main/packages/bpk-react-utils/src/Portal.js#L28-L30


```suggestion
const ACCESSIBILITY_KEYS = {
  Enter: 13,
  Space: 32
};
```

Reply:
Sorted.